PeakCoreData/Core/Operations/CoreDataBatchUpdateOperation.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  CoreDataBatchUpdateOperation.swift
+//  PeakCoreData
+//
+//  Created by David Yates on 21/05/2020.
+//  Copyright © 2020 3Squared Ltd. All rights reserved.
+//
+
+import CoreData
+
+/// Performs an `NSBatchUpdateRequest` and returns an array of updated `NSManagedObjectID`s.
+class CoreDataBatchUpdateOperation<Entity: ManagedObjectType>: CoreDataOperation<[NSManagedObjectID]> {
```

Comment:
This seems handy if you wanna change a simple value. Like a timestamp or a read boolean. I've never used these batch requests because they've always seemed fiddly, so these operations are nice to have.